client/src/components/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -14,27 +61,48 @@ const ProfilePhoto = (): JSX.Element => {
           Profile Photo
         </Typography>
 
-        <Avatar
-          alt="Profile Image"
-          src={`https://robohash.org/${loggedInUser!.email}.png`}
-          className={classes.photoAvatar}
-          variant="circle"
-        />
+        <Avatar alt="Profile Image" src={avatarImage} className={classes.photoAvatar} variant="circle" />
 
         <Typography variant="subtitle1" className={classes.photoText}>
           Be sure to use a photo that clearly shows your face
         </Typography>
         <CardActions>
-          <Button
-            variant="outlined"
-            fullWidth
-            className={classes.photoButton}
-            classes={{
-              root: classes.buttonRoot,
-            }}
-          >
-            Upload a file from your device
-          </Button>
+          <Formik initialValues={initialValues} onSubmit={onFormSubmit}>
+            <form
+              action="/"
+              method="POST"
+              className={classes.form}
+              encType="multipart/form-data"
+              onSubmit={handleSubmit}
+            >
+              {!isInputClicked && (
+                <label htmlFor="imageUpload" className={classes.photoLabel} style={{ borderRadius: '4px' }}>
```

Comment:
You can use the MUI FormLabel here

Ref: https://material-ui.com/api/form-label/

Reply:
used FormLabel